SConstruct

```diff
@@ -1140,18 +1140,27 @@ if require_python or want_python:
         import Cython
         cython_version = LooseVersion(Cython.__version__)
         assert cython_version >= cython_min_version
-    except (ImportError, AssertionError):
-        message = ("Cython not found or incompatible version: "
+    except ImportError:
+        message = "Cython could not be imported."
+        no_cython = True
+    except AssertionError:
+        message = ("Cython is an incompatible version: "
                    "Found {0} but {1} or newer is required.".format(cython_version,
                                                                     cython_min_version))
-        if require_python:
-            print('ERROR: ' + message)
-            sys.exit(1)
-        elif want_python:
-            print('WARNING: ' + message)
-            env['python_package'] = 'minimal'
+        no_cython = True
     else:
-        print('INFO: Using Cython version {0}.'.format(cython_version))
+        no_cython = False
```

Comment:
I might consider `have_cython` as the variable name here, in order to avoid the double negative aspect of this.